docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Control CPU Management Policies on the Node
+---
+
+* TOC
+{:toc}
+
+Kubernetes keeps many aspects of how the pod executes on the node abstracted
```

Comment:
Please remove hard line breaks. From a code perspective, this entire paragraph should be on one line.